MovieQuiz/Presentation/MovieQuizViewController.swift

```diff
@@ -174,5 +157,13 @@ extension MovieQuizViewController {
             showNextQuestion()
         }
     }
+    
+    private func switchEnableForButtons() {
+        [noButton, yesButton].forEach { $0.isEnabled.toggle() }
+    }
+    
+    private func hideBorder() {
+        self.imageView.layer.borderWidth = 0
```

Comment:
Когда есть возможность писать короче, стоит выбирать именно короткую версию
```suggestion
        imageView.layer.borderWidth = 0
```